homeassistant/components/idasen_desk/entity.py

```diff
@@ -0,0 +1,34 @@
+"""Base entity for Idasen Desk."""
+
+from __future__ import annotations
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DeskData, IdasenDeskCoordinator
+
+
+class IdasenDeskEntity(CoordinatorEntity[IdasenDeskCoordinator]):
+    """IdasenDesk sensor."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        unique_id: str,
+        desk_data: DeskData,
+    ) -> None:
+        """Initialize the IdasenDesk sensor entity."""
+        super().__init__(desk_data.coordinator)
+
+        self._attr_unique_id = unique_id
+        self._attr_device_info = dr.DeviceInfo(
+            name=desk_data.device_name,
+            connections={(dr.CONNECTION_BLUETOOTH, desk_data.address)},
+        )
```

Comment:
So I saw that the device_name is now entry.title. If there is a primary device without `name` defined, it will automatically take the entry title. A primary device is a device with more than just `identifiers` or `connections`. So if you add `manufacturer="IKEA"` here, it will become a primary device and you can remove the name

Reply:
Can we add the controller name then?

Reply:
I've set the manufacturer as `LINAK` since they are the controller/motor manufacturer. The official app also shows LINAK on the title.